R/db-interface.R

```diff
@@ -29,6 +29,8 @@
 #'
 #' @family DB interaction functions
 #'
+#' @return A `dm` object on the given `src`
```

Comment:
All sentences should end with a full stop.

```suggestion
#' @return A `dm` object on the given `src`.
```

Reply:
Isn't much of a complete sentence, but alright